src/views/ipv/pyi-escape-m2b.njk

```diff
@@ -26,33 +26,56 @@
 
   <form id="pyiEscapeM2bForm" action="/ipv/page/{{ pageId }}" method="POST">
     <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-    {% set radiosConfig = {
-      idPrefix: "journey",
-      name: "journey",
-      fieldset: {
-        legend: {
-          text: 'pages.pyiEscapeM2b.content.subHeading3' | translate,
-          classes: "govuk-fieldset__legend--m"
-        }
-      },
-      items: [
-        {
-          value: "end",
-          text: 'pages.pyiEscapeM2b.content.formRadioButtons.radioButton1Text' | translate
-        },
-        {
-          value: "next",
-          text: 'pages.pyiEscapeM2b.content.formRadioButtons.radioButton2Text' | translate
-        },
-        {
-          divider: 'pages.pyiEscapeM2b.content.formRadioButtons.separateOptionsInFormText' | translate
-        },
-        {
-          value: "next/bank-account",
-          text: 'pages.pyiEscapeM2b.content.formRadioButtons.radioButton3Text' | translate
-        }
-      ]
-    } %}
+        {% if context === "abandon" %}
```

Comment:
Not delighted with how messy the template is now, thoughts on extracting the radio button as a component?

Reply:
Yeah I think it's a good idea we should start using partials and helpers

Reply:
Yeah can we use a macro or something to tidy this up?